pkg/internal/entrytest/entrytest.go

```diff
@@ -107,11 +107,14 @@ func executeSigning(
 	}
 
 	entrySubmissionChan := make(chan *event.EntrySubmitted)
-	chain.ThresholdRelay().OnRelayEntrySubmitted(
+	_, err = chain.ThresholdRelay().OnRelayEntrySubmitted(
```

Comment:
Let's remove the `error` from the interface of `OnRelayEntrySubmitted`. Let's keep it inside of `ethereum.go`.